src/engines/transit.ts

```diff
@@ -226,6 +228,33 @@ export class TransitVaultClient extends AbstractVaultClient {
      */
     public async decryptText(key: string, ciphertext: string): Promise<string> {
         validateKeyName(key);
-        return this.decrypt(key, { ciphertext }).then((res) => Buffer.from(res.data.plaintext, "base64").toString());
+        return this.decrypt(key, { ciphertext }).then((res) => {
+            if (res.data.plaintext === undefined) {
+                return "";
+            }
+            return Buffer.from(res.data.plaintext, "base64").toString();
+        });
```

Comment:
Since this method is `async` anyway, can we to this without chaining promises, just using `await`?

```suggestion
        const res = await this.decrypt(key, { ciphertext });
        if (res.data.plaintext === undefined) {
            return "";
        }
        return Buffer.from(res.data.plaintext, "base64").toString();
```